src/ui/components/Scanner/Scanner.tsx

```diff
@@ -102,27 +105,20 @@ const Scanner = forwardRef(
       );
     };
 
-    const updateConnections = async (groupId: string) => {
+    const updateConnections = async (
```

Comment:
Hmm, no, this introduces a bug. `setMultiSigGroupCache` should have all of the multi-sig linked contacts. This would keep replacing the array with the latest scanned one (e.g. if you had 5 participants in the multi-sig). We can update that to just append the connection but in a different story.

We only need to be concerned with `updateOrAddMultisigConnectionCache` for now

Reply:
We do, I'm just saying don't make any changes in the `updateConnections` function. Leave it as it was before. It's used for a different purpose than the new redux entry you created